test/extensions/filters/common/ext_authz/ext_authz_http_impl_test.cc

```diff
@@ -346,11 +353,16 @@ using HeaderValuePair = std::pair<const Http::LowerCaseString, const std::string
 // Verify client response headers when authorization_headers_to_add is configured.
 TEST_F(ExtAuthzHttpClientTest, AuthorizationOkWithAddedAuthzHeaders) {
   const auto expected_headers = TestCommon::makeHeaderValueOption(
-      {{":status", "200", false}, {"x-downstream-ok", "1", false}, {"x-upstream-ok", "1", false}});
+      {{":status", "200", envoy::config::core::v3::HeaderValueOption::OVERWRITE_IF_EXISTS_OR_ADD},
+       {"x-downstream-ok", "1",
+        envoy::config::core::v3::HeaderValueOption::APPEND_IF_EXISTS_OR_ADD},
+       {"x-upstream-ok", "1",
+        envoy::config::core::v3::HeaderValueOption::OVERWRITE_IF_EXISTS_OR_ADD}});
```

Comment:
If the test previously passed `false` for all three append values, then I think the new values should all consistently be `OVERWRITE_IF_EXISTS_OR_ADD` to make sure there aren't regressions.